flockoff/validator/database.py

```
@@ -14,6 +15,34 @@ def _init_db(self):
             """CREATE TABLE IF NOT EXISTS miner_scores
                      (uid INTEGER, hotkey TEXT, score REAL, PRIMARY KEY (uid, hotkey))"""
         )
+
+        c.execute(
+            """CREATE TABLE IF NOT EXISTS dataset_revisions
+                       (namespace TEXT PRIMARY KEY, revision TEXT)"""
+        )
+
+        self.conn.commit()
+
+    def get_revision(self, namespace: str) -> typing.Optional[str]:
```

Ruff (0.8.2)

Undefined name `typing`

(F821)